aws_quickstart/datadog_integration_api_call_v2.yaml

```diff
@@ -122,6 +124,7 @@ Resources:
               headers = {
                   'DD-API-KEY': api_key,
                   'DD-APPLICATION-KEY': app_key,
+                  'Dd-Aws-Api-Call-Source': API_CALL_SOURCE_HEADER_VALUE,
```

Comment:
We have this python code in a python file as well in the same directory. Could you add it there as well?

Reply:
I can add it there, but I'm not sure why we have that python code there since it doesn't get executed. I actually excluded from both AFC and organizations.

Reply:
I actually think we should delete the python if we don't need it.

Reply:
Removed these files since they're not used or referenced by any template.